files/u.js/update.json

```diff
@@ -0,0 +1,18 @@
+{
+  "packageManager": "github",
+  "name": "u.js",
+  "repo": "iamso/u.js",
+  "files": {
+  	"include": [
```

Comment:
instead of multiple includes just add `basePath:dist/` and then `include:*.min.*`